plugins/aws/iam/rootSigningCertificate.spec.js

```diff
@@ -0,0 +1,162 @@
+var expect = require('chai').expect;
+const rootSigningCertificate = require('./rootSigningCertificate');
+
+const credentialReports = [
+    {
+      user: '<root_account>',
+      arn: 'arn:aws:iam::560213429563:root',
```

Comment:
is this your account?

Reply:
remember to do a replace all on your private information

Reply:
Updated and noted for future plugins.